backend/src/db/models/ClientCourses.ts

```diff
@@ -8,7 +8,7 @@ const clientCourseSchema = new Schema<IClientCourse>({
   course: { type: Schema.Types.ObjectId, ref: 'Courses' },
   status: { type: String },
   withAssessment: { type: Boolean, default: false },
-  testResult: { type: String },
+  testResult: [{ qN: { type: Number }, aN: { type: Boolean } }],
```

Comment:
Is it really correct to replace this field with another value? Do we need to change any logic on FE?

Reply:
discussed on the call